pym/portage/dbapi/bintree.py

```diff
@@ -568,7 +579,7 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 				metadata[_instance_key(cpv)] = d
 				path = d.get("PATH")
 				if not path:
-					path = cpv + ".tbz2"
+					path = cpv + "." + self.compression["suffix"]
```

Comment:
I can change it to xpak. That will also make sure there's only a single file for a PVR available. We might loose some visibility about the used compression format.